<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="pallavi-PC" name="Test" tests="3" failures="1" timestamp="26 Jul 2019 15:24:12 GMT" time="0.088" errors="0">
  <testcase name="Reports" time="0.029" classname="TestNG.P11"/>
  <testcase name="home" time="0.003" classname="TestNG.P11"/>
  <testcase name="loginscript" time="0.007" classname="TestNG.P11">
    <failure type="java.lang.AssertionError" message="null">
      <![CDATA[java.lang.AssertionError: null
	at TestNG.P11.loginscript(P11.java:13)
	at java.util.concurrent.FutureTask.run(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
... Removed 18 stack frames]]>
    </failure>
  </testcase> <!-- loginscript -->
</testsuite> <!-- Test -->
